README.md

```diff
@@ -317,8 +317,24 @@ For further instructions check out [configlet](https://github.com/exercism/confi
 $ GO111MODULE=off go run exercises/practice/<exercise>/.meta/gen.go
 ```
 
+**NOTE**: If you see the error `json: cannot unmarshal object into Go value of type []gen.Commit` when running the generator you probably have been rate limited by GitHub.
+Try providing a Github access token with the flag `-github_token="<Token>"`.
+Using the token will result in a higher rate limit.
+The token does not need any specific scopes as it is only used to fetch infos about commits.
+
 You should see that some/all of the above files have changed. Commit the changes.
 
+### Synchronizing all exercises with generators
+
+```console
+$ ./bin/run-generators <Github Access Token>
```

Comment:
I am still confused with this part. Let's say my token is 123ABC. Looking at the command here I would expect that I need to run `./bin/run-generators 123ABC`. So far so good. But then the note below says I am supposed to pass the token via `-github_token="<Token>"`. Am I supposed to run `./bin/run-generators 123ABC `-github_token="123ABC"`? If yes, why do I have to provide the token twice?

(Note that I added a commit to your branch, please pull before continuing the work.)